workshops/forms.py

```diff
@@ -346,9 +346,9 @@ class Meta:
         # don't display the 'password', 'user_permissions',
         # 'groups' or 'is_superuser' fields
         # + reorder fields
-        fields = ['username', 'personal', 'middle', 'family', 'may_contact',
-                  'email', 'gender', 'airport', 'affiliation', 'github',
-                  'twitter', 'url', 'notes', 'lessons', 'domains']
+        fields = ['username', 'personal', 'middle', 'family', 'is_active',
+                  'may_contact', 'email', 'gender', 'airport', 'affiliation',
+                  'github', 'twitter', 'url', 'notes', 'lessons', 'domains']
```

Comment:
Alternatively we could add `is_active` to the permissions form, since it's used mostly by admins.